lib/utils.bash

```diff
@@ -384,9 +384,12 @@ repository_needs_update() {
   update_file_dir="$(asdf_data_dir)/tmp"
   local update_file_name
   update_file_name="repo-updated"
-  # `find` outputs filename if it has not been modified in the last day
+  # `find` outputs filename if it has not been modified in the update plugins repository duration.
+  # Default 1 day = 1440 mins
```

Comment:
Can we move this to the documentation?

defaults to `1440`mins (24hrs)

Reply:
I am also happy to change the default to something lower, like 60mins if 1 day is too high